I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 86, in script
    e "Games without menus are called kinetic novels, and there are dozens of them available to play."
Exception: Sayer 'e' is not defined.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 86, in script
    e "Games without menus are called kinetic novels, and there are dozens of them available to play."
  File "C:\Users\elhlyn11_ecolecathol\Downloads\renpy-8.3.2-sdk\renpy\ast.py", line 2573, in execute
    Say.execute(self)
  File "C:\Users\elhlyn11_ecolecathol\Downloads\renpy-8.3.2-sdk\renpy\ast.py", line 575, in execute
    who = eval_who(self.who, self.who_fast)
  File "C:\Users\elhlyn11_ecolecathol\Downloads\renpy-8.3.2-sdk\renpy\ast.py", line 464, in eval_who
    raise Exception("Sayer '%s' is not defined." % who)
Exception: Sayer 'e' is not defined.

Windows-10-10.0.22621 AMD64
Ren'Py 8.3.2.24090902
Lovelorn Reality 1.0
Tue Oct 29 12:45:05 2024
